### PR TITLE
Add option to disable link overlay labels

### PR DESCRIPTION
**Pull request by introt**
Answers #1975

### CODE REVIEW

**Comment by introt** [comment]
> Preference "Preview link destination in tooltip when hovered over or under cursor" not documented in manual page

Ah, forgot about this. Will push fixup after #1981 gets merged

**Comment by introt** [comment]
Ready for review, would like some feedback on the wording/terms.

**Comment by jaap-karssenberg** [comment]
Could you rephrase the option name in the GUI to be a bit shorter? E.g. "Show link destination in overlay"

**Comment by introt** [comment]
Done; went with "Preview" as it does resolve absolute paths
